### DISCUSSION TITLE
Issue with output of `crowd.navMeshQuery.queryPolygons`

### DISCUSSION THREAD

**Discussion by rob-myers** [isaac-mason/recast-navigation-js#444] [General]
Seems `crowd.navMeshQuery.queryPolygons(center, halfExtent)` has output like:
```js
[4485126, 0, 0, ...]
```
How do we distinguish the case where `0` is the ref of one of the polygons you want?
In my particular use case `0` is a valid polygon ref.

**Comment by isaac-mason** [comment]
Hey @rob-myers, I believe right now you need to use the returned `polyCount` value to determine which results are populated, and which can be ignored.

The `polyCount` is returned by the wasm method, but we could definitely change the wrapped queryPolygons method to just not return those polys.

**Reply by isaac-mason** [reply]
https://github.com/isaac-mason/recast-navigation-js/commit/753e4e2fb8dd2ac4a7a5bb7d01d71e312b1adc9b

**Reply by rob-myers** [reply]
Much appreciated!
